tests/integration/integration_tests_plugin.py

```diff
@@ -30,6 +30,7 @@ def output_file_url(request, file_server):
     filename = request.function.__name__ + '.html'
     file_obj = request.fspath.dirpath().join(filename)
     file_path = file_obj.strpath
+    url = file_path.replace('\\', '/')  # Windows-proof
```

Comment:
This is not the place to do this.

The place would be to fix handling of windows file_paths in `tests/plugins/file_server.py` `where_is` method

https://github.com/bokeh/bokeh/blob/almarklein/4110-resize/tests/plugins/file_server.py#L115